src/Core/Bucketer.php

```diff
@@ -157,15 +157,16 @@ public static function getCampaignUsingRange($rangeForCampaigns, $campaigns)
      *
      * @param  $userId
      * @param  $campaign
-     * @param  bool $disableLogs optional: disable logs if True
+     * @param  bool  $is_new_bucketing_enabled
+     * @param  bool  $disableLogs optional: disable logs if True
      * @return array returns bucket value and hash value
      */
     /*
     * The function getBucketVal has references from “Optimizely PHP SDK, version 3.2.0” Copyright 2016-2019, Optimizely,
     * Copyright 2016-2019, Optimizely, used under Apache 2.0 License.
     * Source - https://github.com/optimizely/php-sdk/blob/master/src/Optimizely/Bucketer.php
     */
-    public static function getBucketVal($userId, $campaign = [], $is_new_bucketing_enabled, $disableLogs = false)
+    public static function getBucketVal($userId, $campaign = [], bool $is_new_bucketing_enabled = true, $disableLogs = false)
```

Comment:
I believe this `bool` should be removed since the type declaration for booleans is not available as of php5.6
```suggestion
    public static function getBucketVal($userId, $campaign = [], $is_new_bucketing_enabled = true, $disableLogs = false)
```

Reply:
Removed the type declaration1